Assets/AppCenter/Editor/AppCenterPreBuild.cs

```diff
@@ -36,12 +36,16 @@ public void OnPreprocessBuild(BuildTarget target, string path)
             {
                 FirebaseDependency.SetupPush();
             }
+#if !APPCENTER_DONT_USE_NATIVE_STARTER
```

Comment:
Consider change the macro to `APPCENTER_USE_NATIVE_STARTER`

Reply:
@jaelim-ms disagree on this one. This is an optional directive for a very small fraction of our customers who want to delay the start. So if we change to `APPCENTER_USE_NATIVE_STARTER`, almost everyone will have to specify it.